gcip/core/include.py

```diff
@@ -13,19 +65,52 @@
 
 
 class Include(metaclass=ABCMeta):
+    """
+    This is just an abstract superclass.
+
+    Please use one of the subclasses:
+
+    * `IncludeLocal`
+    * `IncludeFile`
+    * `IncludeRemote`
+    * `IncludeTemplate`
+    * `IncludeArtifact`
+    """
 
     _rendered_include: Dict[str, str]
 
     def render(self) -> Dict[str, str]:
```

Comment:
```suggestion
    def render(self) -> Dict[str, Any]:
```

Reply:
`_rendered_include` is of `Dict[str, str]`. But you're right - I provided the wrong type in the documentation. I'll fix that.